cfme/utils/log_validator.py

```diff
@@ -85,28 +110,22 @@ def matches(self):
         logger.info("Matches found: {}".format(self._matches))
         return self._matches
 
-    def validate_logs(self):
-        """Validate log pattern"""
-        for pattern, count in self.matches.items():
-            if count == 0:
-                pytest.fail(
-                    'Expected pattern {} did not match; match count {}'.format(pattern, count)
-                )
-
-    def wait_for_log_validation(
-            self, delay=5, num_sec=180, message="waiting for log validation", **kwargs
-    ):
-        """ Wait for log validation, takes the kwargs as wait_for. This function will reduce
-            duplicate functions in tests that wait_for log_validation. It is necessary to create
-            this function since _verify_match_logs raise pytest.fail() when it fails to find the
-            match pattern in the logs.
+    def validate(self, wait=None, message="waiting for log validation", **kwargs):
+        """Validate log pattern
 
-            Note that you must call fix_before_start() before making use of this function.
+        Args:
+            wait: Wait for log validation (timeout)
+            message: Specific message.
+        Returns (bool): True if expected pattern matched in log else False
```

Comment:
I'm not sure I understand when it will return False.

Reply:
right actually I was returning an output of `func` which was wrong.
Now in case of wait, it will return `True` if wait_for success else `TimedOutError` and without wait it will return `bool`